flink-python/docs/reference/pyflink.table/expressions.rst

```diff
@@ -230,6 +230,7 @@ advanced type helper functions
     Expression.array_position
     Expression.array_remove
     Expression.array_reverse
+    Expressions.array_slice
```

Comment:
```suggestion
    Expression.array_slice
```
Otherwise i have no idea why it sohuld be `Expressions` while everyone else is just `Expression`